app/services/classification/base.rb

```diff
@@ -29,7 +29,7 @@ def class_parameters
     def possible_value_orders
       class_parameters.select do |key|
         # take only keys with actual values
-        key.lookup_values_count > 0 # we use counter cache, so its safe to make that query
+        key.lookup_values.count > 0 # we use counter cache, so its safe to make that query
```

Comment:
Is this comment still true, since it no longer uses the counter cache?  Is there anything "unsafe" here?